### PR TITLE
feat: resolve issue #23

### PR DESCRIPTION
**Pull request by Gatekeeper2004**

### CODE REVIEW

**Comment by livid** [comment]
Awesome, I will test it ASAP. Thanks!

**Comment by livid** [comment]
@wyd2004 I clicked "Show" from the top menu and got this:

Screen Shot 2022-06-08 at 10 13 11 PM

File: Info.plist [hunk 1]
```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict/>
```
Comment (livid) [inline Info.plist]: URL types seem empty.

Comment (Gatekeeper2004) [reply Info.plist]: Fixed.  My negligence.

**Comment by livid** [comment]
Thanks for the new feature. I have merged it.

This feature reminds me of another issue:

Currently, it seems that SwiftUI cannot remember window size and position if you closed it with CMD+W. It can only remember if you do CMD+Q.

**Pull request merged**